cli/store_cmds.go

```diff
@@ -309,7 +317,8 @@ func Import(buildDataFS vfs.FileSystem, stor interface{}, opt ImportOpt) error {
 				return importGraphData(rule.Target(), rule.Unit)
 			})
 		case *grapher.GraphMultiUnitsRule:
-			for target, sourceUnit := range rule.Targets() {
+			for target_, sourceUnit_ := range rule.Targets() {
+				target, sourceUnit := target_, sourceUnit_
```

Comment:
I see, I followed the `rule := rule_` style from the enclosing loop; I don't feel strongly about either way.